lib/rubocop/cop/rails/unknown_env.rb

```diff
@@ -86,7 +86,9 @@ def unknown_env_name?(name)
         end
 
         def environments
-          cop_config['Environments']
+          e = cop_config['Environments']
+          e += ['local'] if target_rails_version >= 7.1
+          e
```

Comment:
Worth memoizing this to create less _garbage_? Otherwise, we'll create a new throwaway array every time this is called (possibly several times per occurrence of `Rails.env...`)? I'm not sure when or how often cops are instantiated though, which would affect how much value there is to memoizing it. 🤷‍♂️

Reply:
I think so. Done 👍